model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -83,6 +83,13 @@ data class AnalyzerResult(
             }
         }
 
+        packages.forEach { curatedPackage ->
```

Comment:
Commit message nit: Maybe start the sentence with "Add an error if ...".

Reply:
Done.